applications/incompressible_navier_stokes/taylor_green_vortex/application.h

```diff
@@ -102,7 +102,7 @@ class Application : public ApplicationBase<dim, Number>
 
     // clang-format off
     prm.enter_subsection("Application");
-      prm.add_parameter("MeshType",        mesh_type_string,                  "Type of mesh (Cartesian versus curvilinear).", dealii::Patterns::Selection("Cartesian|Curvilinear"));
+      prm.add_parameter("MeshType", mesh_type, "Type of mesh (Cartesian versus curvilinear).") ;
```

Comment:
We would need to fix indentation manually here.

Should probably be `Curvilinear` (capital "C") to avoid misunderstandings?!

Reply:
[fe22560](https://github.com/exadg/exadg/pull/382/commits/fe22560ec03fbd3134afc96a469110348093d320)